operations/automated_upgrades/README.markdown

```diff
@@ -0,0 +1,143 @@
+# Consul Enterprise Automated Upgrades Guide
```

Comment:
Could you take a look at the [Guide template](https://docs.google.com/document/d/17zk1a_azbAp0ph4_D_xh8E-LJ6tIFR_5C967ik1Rj9c/edit) and see if you can modify the README to follow that structure? I understand that it may be necessary to deviate from the template at times, but I'd prefer to use that as a starting point so we have a common flow to our guides. Let me know if you think that structure won't satisfy this guide.

At the very least we should have the business case, challenge, & solution up front so people know why they should even complete the guide.

Reply:
ACK

Reply:
❤️